bgpd/bgpd.c

```diff
@@ -2738,6 +2738,9 @@ static struct bgp *bgp_create(as_t *as, const char *name,
 	struct bgp *bgp;
 	afi_t afi;
 	safi_t safi;
+	struct utsname names;
```

Comment:
Sure, make sense. Will define API in command.c/command.h (thats where all the host info seems to be present/modified) and use those APIs in BGPD.